nixos/modules/virtualisation/qemu-vm.nix

```diff
@@ -802,13 +799,28 @@ in
         default = true;
         description =
           lib.mdDoc ''
-            If enabled, the boot disk of the virtual machine will be
+            If enabled, the system disk of the virtual machine will be
             formatted and mounted with the default filesystems for
             testing. Swap devices and LUKS will be disabled.
 
             If disabled, a root filesystem has to be specified and
             formatted (for example in the initial ramdisk).
           '';
+        };
+
+    virtualisation.useDefaultPartitionTable =
```

Comment:
I think it would be more elegant to add an `partitionTableLayout` option which has a default value. That would make the module more composable.